### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by husaynhakeem**
This PR mostly corrects typos and makes minor syntax improvements.

### CODE REVIEW

**Review by weeeBox — APPROVED** [review]
LGTM

**Comment by husaynhakeem** [comment]
Great. I don't have write access, so I can't merge the commit. Feel free to merge it yourself.

**Pull request merged**
